### PR TITLE
Fix outdated / incorrect links in bahasa Indonesia localization guide

### PR DESCRIPTION
**Pull request by sftim**
Fix two incorrect hyperlinks in https://kubernetes.io/id/docs/contribute/localization_id/

/language id

### CODE REVIEW

File: content/id/docs/contribute/localization_id.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ ditulis dalam huruf kapital pada halaman asli bahasa Inggris.
 
 ### Panduan untuk "Feature Gate" Kubernetes
 
-Istilah [_functional gate_](https://kubernetes.io/ko/docs/reference/command-line-tools-reference/feature-gates/) 
+Istilah [_functional gate_](/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (sftim) [inline content/id/docs/contribute/localization_id.md:133]: BTW https://kubernetes.io/id/docs/reference/command-line-tools-reference/feature-gates/ doesn't exist yet.

File: content/id/docs/contribute/localization_id.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ ditulis dalam huruf kapital pada halaman asli bahasa Inggris.
 
 ### Panduan untuk "Feature Gate" Kubernetes
 
-Istilah [_functional gate_](https://kubernetes.io/ko/docs/reference/command-line-tools-reference/feature-gates/) 
+Istilah [_functional gate_](/docs/reference/command-line-tools-reference/feature-gates/) 
 Kubernetes tidak perlu diterjemahkan ke dalam bahasa Indonesia dan tetap 
 dipertahankan dalam bentuk aslinya.
```
Comment (jihoon-seo) [inline content/id/docs/contribute/localization_id.md:135]: [id]
> Istilah "functional gate" Kubernetes tidak perlu diterjemahkan ke dalam bahasa Indonesia dan tetap dipertahankan dalam bentuk aslinya.

[English translation]
> The term "functional gate" Kubernetes does not need to be translated into Indonesian and is preserved in its original form.

---

What about replacing "functional gate" with "feature gate"?

```suggestion
Istilah [_feature gate_](/docs/reference/command-line-tools-reference/feature-gates/) 
Kubernetes tidak perlu diterjemahkan ke dalam bahasa Indonesia dan tetap 
dipertahankan dalam bentuk aslinya.
```

**Comment by kbhawkey** [comment]
Ping id team for approval, /cc @kubernetes/sig-docs-id-reviews

**Pull request merged**